taprpc/mintrpc/mint.proto

```diff
@@ -124,6 +124,9 @@ message UnsealedAsset {
 
     // The group virtual transaction for the asset.
     taprpc.GroupVirtualTx group_virtual_tx = 3;
+
+    // The byte serialized PSBT equivalent of the group virtual transaction.
+    string group_virtual_psbt = 4;
```

Comment:
`SealBatch` can also optionally accept this.

Reply:
This PR will handle that change: https://github.com/lightninglabs/taproot-assets/pull/1295